app/containers/InterventionDetailsPage/containers/ParticipantInviter/CopyLinkForm.tsx

```diff
@@ -49,6 +50,10 @@ export const CopyLinkForm: FC<Props> = ({
     [isModularIntervention, isReportingIntervention],
   );
 
+  const filteredSessionOptions = sessionOptions.filter(
+    (session) => session.type !== 'Session::Sms',
+  );
```

Comment:
I would do that filtering in `app/containers/InterventionDetailsPage/containers/ParticipantInviter/InviteParticipantsModalContent.tsx` when creating sessionOptions
and revert changes in this file. This component is not responsible for creating session options, it only receives them.